src/function_body.rs

```diff
@@ -1,13 +1,98 @@
 use backend::*;
 use error::Error;
-use wasmparser::{FunctionBody, Operator};
+use wasmparser::{FunctionBody, Operator, Type};
+
+// TODO: Use own declared `Type` enum.
+
+/// Type of a control frame.
+#[derive(Debug, Copy, Clone, PartialEq)]
+enum ControlFrameKind {
+    /// A regular block frame.
+    ///
+    /// Can be used for an implicit function block.
+    Block { end_label: Label },
+    /// Loop frame (branching to the beginning of block).
+    #[allow(unused)]
```

Comment:
Is this `#[allow(unused)]` still needed?

Reply:
Yeah because we don't construct it anywhere.

Reply:
Ah, right. I forget how smart the Rust compiler is sometimes :-).